### PR TITLE
[feat]: [DBOPS-306]: Onboard DBDevOps entities

### PR DESCRIPTION
**Pull request by Abhishek-CDC**
## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- gitleaks: `trigger gitleaks`

### CODE REVIEW

File: harness/dbops/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+*.cgo2.c
+_cgo_defun.c
+_cgo_gotypes.go
+_cgo_export.*
+
+_testmain.go
+
+*.exe
+*.test
+*.prof
```
Comment (yogesh-chauhan) [inline harness/dbops/.gitignore:24]: is this file needed ?

File: harness/dbops/.swagger-codegen-ignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# You can recursively match patterns against a directory, file or extension with a double asterisk (**):
+#foo/**/qux
+# This matches foo/bar/qux, foo/baz/qux, and foo/bar/baz/qux
+
+# You can also negate patterns with an exclamation (!).
+# For example, you can ignore all files in a docs folder with the file extension .md:
+#docs/*.md
+# Then explicitly reverse the ignore rule for a single file:
+#!docs/README.md
```
Comment (yogesh-chauhan) [inline harness/dbops/.swagger-codegen-ignore:23]: is this file needed ?

File: harness/dbops/.swagger-codegen/VERSION [hunk 3]
```diff
@@ -0,0 +1 @@
+3.0.57
```
Comment (yogesh-chauhan) [inline harness/dbops/.swagger-codegen/VERSION:1]: is this file needed ?

File: harness/dbops/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+language: go
+
+install:
+  - go get -d -v .
+
+script:
+  - go build -v ./
```
Comment (yogesh-chauhan) [inline harness/dbops/.travis.yml:7]: is this file needed ?

File: harness/dbops/api/swagger.yaml
Comment (yogesh-chauhan) [inline harness/dbops/api/swagger.yaml:1]: do we need to check in swagger.yaml also ?

File: harness/dbops/api_pipeline_step.go
Comment (yogesh-chauhan) [inline harness/dbops/api_pipeline_step.go:1]: We can remove this API

File: harness/dbops/api_log_ingest.go
Comment (yogesh-chauhan) [inline harness/dbops/api_log_ingest.go:1]: We can remove this API

File: harness/dbops/git_push.sh [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        git remote add origin https://${git_user_id}:${GIT_TOKEN}@github.com/${git_user_id}/${git_repo_id}.git
+    fi
+
+fi
+
+git pull origin master
+
+# Pushes (Forces) the changes in the local repository up to the remote repository
+echo "Git pushing to https://github.com/${git_user_id}/${git_repo_id}.git"
+git push origin master 2>&1 | grep -v 'To https'
```
Comment (yogesh-chauhan) [inline harness/dbops/git_push.sh:51]: do we need this file?

**Review by yogesh-chauhan — CHANGES_REQUESTED** [review]

File: harness/dbops/.gitignore (hunk 1 above)
Comment (Abhishek-CDC) [reply harness/dbops/.gitignore:24]: present in 'nextgen' folder in same parent directory as well

File: harness/dbops/.swagger-codegen-ignore (hunk 2 above)
Comment (Abhishek-CDC) [reply harness/dbops/.swagger-codegen-ignore:23]: present in 'nextgen' folder in same parent directory as well

File: harness/dbops/.swagger-codegen/VERSION (hunk 3 above)
Comment (Abhishek-CDC) [reply harness/dbops/.swagger-codegen/VERSION:1]: present in 'nextgen' folder in same parent directory as well

File: harness/dbops/.travis.yml (hunk 4 above)
Comment (Abhishek-CDC) [reply harness/dbops/.travis.yml:7]: present in 'nextgen' folder in same parent directory as well

File: harness/dbops/api/swagger.yaml
Comment (Abhishek-CDC) [reply harness/dbops/api/swagger.yaml:1]: present in 'nextgen' folder in same parent directory as well

File: harness/dbops/git_push.sh (hunk 5 above)
Comment (Abhishek-CDC) [reply harness/dbops/git_push.sh:51]: present in 'nextgen' folder in same parent directory as well

File: harness/dbops/api_log_ingest.go
Comment (Abhishek-CDC) [reply harness/dbops/api_log_ingest.go:1]: Done

File: harness/dbops/api_pipeline_step.go
Comment (Abhishek-CDC) [reply harness/dbops/api_pipeline_step.go:1]: Done

**Review by yogesh-chauhan — APPROVED** [review]
